plugins/scanners/fossid/src/main/kotlin/FossIdConfig.kt

```diff
@@ -131,7 +131,11 @@ data class FossIdConfig(
 
     /** Whether to write scan results to the storage. */
     @OrtPluginOption(defaultValue = "true")
-    val writeToStorage: Boolean
+    val writeToStorage: Boolean,
+
+    /** Treats pending identifications as errors instead of hints. */
```

Comment:
`Treats` should be `Treat`. This docs will be used for the [documentation on the website](https://oss-review-toolkit.org/ort/docs/plugins/scanners/FossID).